store.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Karma</title>
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+</head>
+
+<header id="header">
+    <img class ="logo"src="img/karma-logo.svg" alt="logo"  width="40px" height= "40px"/>
+
+<nav class ="navBar">
+    <a class="active" href="#Meet Karma">Meet Karma</a>
+    <a href="#How it Works">How it Works</a>
+    <a href="#Store">Store</a>
+    <a href="#Blog">Blog</a>
+    <a href="#Help">Help</a>
+    <a href="#Login">Login</a>
+      
+</nav>
+</header>
+<br><br><br>
+
+<body>
+<div class ="container">
+  <h1> Order your Karma WiFi today!</h1>
+  <form action="#">
+<div class="userdetails">
+
+  <div class="inputbox">
+    <span class="details"> Full Name *</span>
+<input type="text" placeholder="" required>
+  </div>
+  <div class="inputbox">
+    <span class="details"> Last Name *</span>
+<input type="text" placeholder="" required>
+  </div>
+  <div class="inputbox">
+    <span class="details"> Address *</span>
+<input type="text"  placeholder="" width="700px" required>
+  </div>
+  <br><br>
+  <div class="inputbox">
+    <span class="details"> Address 2</span>
+<input type="text" placeholder="" required>
+  </div>
+
+  <label for="dropdown" id="city" style="width: 90vw;"> City *</label>
+  <select id="dropdown">
+    <option value = "City-1"> London</option>
+    <option value = "City-2"> Birmingham</option>
+    <option value = "City-3"> Manchester</option>
+    <option value = "City-4"> Leeds</option>
+    <option value = "City-5"> Bath</option>
+  </select>
+<br>
+<br>
+
+  <div class="inputbox">
+    <span class="details"> Postcode *</span>
+<input type="text" placeholder="" required>
+  </div>
+
+<div class="colordetails">
+  <span class="colortitle">Select a color</span>
+  <div class ="category">
+    <label for="color" id="color"> <b>Karma Orange:</b> </label>
+    <br>
+      <input id="color" type="radio" name="color" value="black">
+      <label for="color"> Space Grey </label>
+<br><br><br><br><br>
+<input id="submit" type="submit" value="Place Order" class="submit-button">
+</div>
+</div>
+    
+</div>
+</div>
```

Comment:
Please fix the indentation here. 
In VS Code, try `F1` and then  "Format Document".